alica_tests/src/test/Expr/include/NavigateToPick.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
I guess this branch is before blackboard support, otherwise can use a single navigation behaviour with a parameter specifying the target

Reply:
It already has an older version of the blackboard. However in setParameters, there is no way to access the worldModel, as the PlanAttachment object, on which the setParameters method is called, does not store any ptr to the worldModel / engine and takes neither as an input to setParameters.